### PR TITLE
Weighted cross entropy loss

### PR DESCRIPTION
**Pull request by rlacombe**
- Hotfixed weighted cross entropy loss weighted by prevalence of each category to deal with dataset imbalance
- Detailed confusion metrics
- Configs new models

### CODE REVIEW

File: climatenet/utils/utils.py [hunk 1]
```diff
@@ -42,9 +42,8 @@ def __init__(self, path: str):
         self.labels = self.config_dict['labels']
         self.description = self.config_dict['description']
         self.loss = self.config_dict['loss']
-
-        if 'weights' in self.config_dict: self.weights = self.config_dict['weights']  # TODO: raise exception if None
-        self.save_epochs = self.config_dict.get('save_epochs', False) 
+        self.loss_weights = self.config_dict['loss_weights'] if 'loss_weights' in self.config_dict else [] # TODO: raise exception if None
```
Comment (hannahg141) [inline climatenet/utils/utils.py:45]: don’t think we need this anymore

**Review by hannahg141 — APPROVED** [review]
